content/en/blog/2024/otel-collector-survey/index.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Insights from the OpenTelemetry Collector Survey
+linkTitle: OpenTelemetry Collector Survey
+date: 2024-05-01
```

Comment:
```suggestion
date: 2024-05-08
```

Let's aim for tomorrow, we just released the collector roadmap blog post yesterday